File: langs/typescript/problems/load_balancer/stub.ts
class Simulation {
  constructor() {}
  addBackend(backendId) { throw new Error('not implemented'); }
  route() { throw new Error('not implemented'); }
  setHealth(backendId, flag) { throw new Error('not implemented'); }
  setWeight(backendId, weight) { throw new Error('not implemented'); }
  sticky(clientId) { throw new Error('not implemented'); }
  done(backendId) { throw new Error('not implemented'); }
}
module.exports = { Simulation };

File: langs/ruby/problems/load_balancer/stub.rb
class Simulation
  def initialize
  end
  def add_backend(backend_id)
    raise 'not implemented'
  end
  def route()
    raise 'not implemented'
  end
  def set_health(backend_id, flag)
    raise 'not implemented'
  end
  def set_weight(backend_id, weight)
    raise 'not implemented'
  end
  def sticky(client_id)
    raise 'not implemented'
  end
  def done(backend_id)
    raise 'not implemented'
  end
end
